RESideMenu/RESideMenuController.m

```diff
@@ -395,6 +395,8 @@ - (void)buildLayersForAnimation
     transform.m34 = -1.0 / (contentView.bounds.size.height * 4.6666667);
 
     self.perspectiveAnimationLayer = animationLayer;
+    self.perspectiveAnimationLayer.shouldRasterize = YES;
+    self.perspectiveAnimationLayer.rasterizationScale = [UIScreen mainScreen].scale;
```

Comment:
`perspectiveAnimationLayer` only contains an image snapshot of the map view anyway, so the behaviour should be unchanged from what we have currently. What this change really does is tells Core Animation to render a bit map of the layer before compositing, which gives us the built in anti-aliasing.